### PR TITLE
Truncate container commands and probes for the entire content not each arg

### PR DESCRIPTION
**Pull request by jwforres**
Fixes https://github.com/openshift/origin-web-console/issues/1009

### CODE REVIEW

File: app/scripts/filters/resources.js [hunk 1]
```diff
@@ -1322,8 +1322,8 @@ angular.module('openshiftConsole')
       return cmd;
     };
 
-    return function(container, image) {
-      if (!container || !image) {
+    return function(container, image, returnArgsIfNoCmd) {
```
Comment (spadgett) [inline app/scripts/filters/resources.js]: Should `returnArgsIfNoCmd` just always be true? We should update debug terminal to set that, and that's the only other place the filter is used..

Comment (jwforres) [inline app/scripts/filters/resources.js]: Originally I wasn't adding `<image-entrypoint>` when we didn't have the image and it was only returning the args so I didn't want it to change the behavior on debug terminal. But yeah now I think I can drop the arg and just always have that behavior.

**Comment by jwforres** [comment]
@spadgett updated

**Comment by spadgett** [comment]
[merge]

**Pull request merged**